config/nonprod.tfvars

```
@@ -0,0 +1,13 @@
+env = "nonprod"
+subusers = [
```

I would just call this accounts as it's probably clearer
```suggestion
accounts = [
```

---

done